src/main.rs

```diff
@@ -155,17 +155,33 @@ impl Fixer {
             .functions()
             .filter_map(|function| {
                 let function = function.ok()?;
+                // This eprintln! is useful when creating tests.
+                //eprintln!(
+                //    "FILE 0x{:x} size={} func={}",
+                //    function.address,
+                //    function.size,
+                //    function.name.as_str()
+                //);
                 Some(FuncInfo {
                     address: function.address,
                     size: function.size,
                     mangled_name: function.name.as_str().to_string(),
                     line_infos: function
                         .lines
                         .into_iter()
-                        .map(|line| LineInfo {
-                            address: line.address,
-                            line: line.line,
-                            path: interner.intern(line.file.path_str()),
+                        .map(|line| {
+                            // This eprintln! is useful when creating tests.
+                            //eprintln!(
+                            //    "LINE 0x{:x} line={} file={}",
+                            //    line.address,
+                            //    line.line,
+                            //    line.file.path_str()
+                            //);
```

Comment:
Why the comment is repeated twice? (the other one is above at line 158)

Reply:
Because it's true for both cases :)  I added an "also" to the second comment, to make it read better in combination with the first comment.